guides/overview/app-structure.md

```diff
@@ -28,36 +28,36 @@ only.
 # `config/`
 
 Holds the declarative and executable configuration for your app. *.js files
-(other than `environment.js` and `initializers/*`) in this folder will be
+(other than `environment.js` and `initializers/*.js`) in this folder will be
 automatically loaded into the container under their filename (i.e.
 `config/foo.js` will be loaded under `config:foo`).
 
 Configuration files in Denali tend to be JS files that export a function which
 returns the configuration object. This gives the app developer greater
 flexibility in constructing configuration: it allows for inter-dependent values
-(i.e. option a depends on the value of option b), and easily using enviornment
+(i.e. option A depends on the value of option B), and easily using environment
 variables via `process.env`.
 
 Third party addons can add their own configuration files this way as well.
 
 ## `config/environment.js`
 
-This file holds the configuration that varies per environment (i.e. dev vs.
-staging. vs prod database details).
+This file holds the configuration that varies per environment (i.e. development vs.
+staging vs. production database details).
 
 ## `config/middleware.js`
 
 This file exports a function which is invoked with the application's Router as
-it's first argument. It lets you add generic Connect-compatible middleware to
+its first argument. It lets you add generic Connect-compatible middleware to
 your application that will run _before_ the Router hands off to the appropriate
 action.
 
 ## `config/routes.js`
 
 You define your application's routes in this file. See the [Routing
-guide](latest/guides/application/routing) for more details.
+guide](../../application/routing) for more details.
```

Comment:
Note that I changed the paths here and below so that we don't run into issues with these links if/when we have multiple versions of the guides.  We wouldn't want v1.0 links linking to Canary pages necessarily ...